### ISSUE TITLE
Driftapi nicht mehr nutzbar

### ISSUE THREAD

**Issue by GeneSysTobiasWagner** [christiangeissler/driftapi#17]
Verstehe ich das aus den Forenbeiträgen richtig, dass die Driftapi quasi nicht mehr nutzbar ist, da man die App nicht mehr verbunden bekommt aufgrund einiger Zertifikats und HTTPS Themen?

**Comment by christiangeissler** [comment]
Nein, das ist so nicht richtig. Die API funktioniert weiterhin, sonst würde z.B. driftclub auch nicht funktionieren. Die Server die über die driftapi erreichbar sind, müssen allerdings https unterstützen. Die Referenzimplementierung hier auf der Webseite, die ja für lokale Netzwerke ausgelegt ist und kein https unterstützt, funktioniert also aktuell nicht. Aber wenn du selbst einen Webserver aufsetzt und die api via https anbietest, geht alles.
